javascript-light-bulb/main.js

```diff
@@ -0,0 +1,19 @@
+function instanceChange() {
+  const sphereColumn = document.querySelector('.sphere-column');
+
+  sphereColumn.addEventListener('click', function onClick(event) {
+    sphereColumn.classList.toggle('instance-change');
+  });
+}
+
+instanceChange();
+
+function backgroundChange() {
+  const background = document.querySelector('.page-container');
+
+  background.addEventListener('click', function onClick(event) {
+    background.classList.toggle('background-change');
+  });
+}
+
+backgroundChange();
```

Comment:
A couple things here.

1. We should condense this into 1 event listener, we don't need 2 different ones. We can accomplish this with only 1 event listener on the button, and adjust the classes inside there.
2. Adding eventListeners should be done globally rather than inside a function. The problem with your current approach, is that every-time this function gets called, you are adding another event listener to the element. So your button can now have like 20 event listeners attached to it. If you do it globally, then only one event listener will be attached to it.

Reply:
Thanks for the tips! I was able to condense the code, let me know if it everything checks out now. Thanks!